_includes/footer.html

```diff
@@ -46,8 +46,8 @@ <h3 class="widget-title">Support</h3>
               <dd><a href="https://fedoraproject.org/wiki/Communicating_and_getting_help">Get Help</a></dd>
               <dd><a href="https://ask.fedoraproject.org/">Ask Fedora</a></dd>
               <dd><a href="https://fedoraforum.org/">Forums</a></dd>
-              <dd><a href="https://fedoraproject.org/wiki/Common_F25_bugs">Common Bugs</a></dd>
-              <dd><a href="https://docs.fedoraproject.org/en-US/Fedora/25/html/Installation_Guide">Installation Guide</a></dd>
+              <dd><a href="https://fedoraproject.org/wiki/Common_F{{ site.fedora_release }}_bugs">Common Bugs</a></dd>
+              <dd><a href="https://docs.fedoraproject.org/en-US/Fedora/{{ site.fedora_release }}/html/Installation_Guide">Installation Guide</a></dd>
```

Comment:
This does not seem to be pointing to the right site (I am getting 404).

This one should be the right one.
```suggestion
              <dd><a href="https://docs.fedoraproject.org/en-US/fedora/f{ site.fedora_release }}/install-guide/">Installation Guide</a></dd>
```